controller/eventQuestionController.js

```diff
@@ -8,6 +9,15 @@ const StatusEnum = require('../model/enums').StatusEnum;
 
 const create = (req, res) => {
 
+    const user = req.decoded;
+
+    if (!user || !ValidationUtil.isUserAdmin(user.accessType) || !req.body) {
```

Comment:
Aren't regular users supposed to create the event questions, by this check, only admins can create questions.

Reply:
You are correct.